app/Recipes/Breeze/ApiOnly.php

```diff
@@ -58,6 +58,9 @@ public function __invoke(): void
                 'boot',
                 "ResetPassword::createUrlUsing(function (object \$notifiable, string \$token) {\n    return config('app.frontend_url').\"/password-reset/\$token?email=\{\$notifiable->getEmailForPasswordReset()}\";\n});"
             );
+            $step->formatter->file('app/Providers/AppServiceProvider.php')->rules('ordered_imports')->fix();
+            // or an array of rules: $step->formatter->file('app/Providers/AppServiceProvider.php')->rules(['ordered_imports'])->fix();
+            // or use the builder: $step->formatter->file('app/Providers/AppServiceProvider.php')->importOrder()->fix();
```

Comment:
I mentioned this in a chat, but I think my favorite is just:

```php
$step->formatter->file('whatever')->fix('string or array of strings');
```

Reply:
I implemented:

$step->formatter->fix('whatever', 'string or array of strings');

Instead of:

$step->formatter->file('whatever')->fix('string or array of strings');

The reason is that all our other step tools don't have state that is maintained between tool usages. I was worried that:

$step->formatter->file('whatever')->fix('rule'); then $step->formatter->fix('rule'); later in the recipe would be totally valid but act on the wrong file.

Reply:
I think that's super reasonable! Good call!

Reply:
If we *did* want that syntax, we could have `file('')` return a temporary object, maybe like `FileFormatter`, which contained the `fix()` method. But I think the current syntax is good!